src/main/java/com/diamondfire/helpbot/bot/events/ChannelUpdatedNameEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.diamondfire.helpbot.bot.events;
+
+import com.diamondfire.helpbot.bot.HelpBotInstance;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.channel.forums.ForumTag;
+import net.dv8tion.jda.api.events.channel.update.*;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
+
+public class ChannelUpdatedNameEvent extends ListenerAdapter {
+    
+    @Override
+    public void onChannelUpdateName(@NotNull ChannelUpdateNameEvent event) {
+        // Limit to help forum.
+        if (
+                event.getChannel().getType() != ChannelType.GUILD_PUBLIC_THREAD ||
+                        event.getChannel().asThreadChannel().getParentChannel().getIdLong() != HelpBotInstance.getConfig().getHelpChannel()
+        ) {
+            return;
+        }
+        
+        ThreadChannel threadChannel = event.getChannel().asThreadChannel();
+        ForumTag solvedTag = threadChannel.getParentChannel().asForumChannel().getAvailableTagById(HelpBotInstance.getConfig().getHelpChannelSolvedTag());
+        
+        if (event.getOldValue() == null || event.getNewValue() == null) return;
+        
+        System.out.println();
+        System.out.println(event.getOldValue());
+        System.out.println(event.getNewValue());
```

Comment:
are these meant to be here

Reply:
guh i got distracted before i got to remove them